cmd/ctr/commands/tasks/tasks_unix.go

```diff
@@ -51,12 +51,12 @@ func HandleConsoleResize(ctx gocontext.Context, task resizer, con console.Consol
 }
 
 // NewTask creates a new task
-func NewTask(ctx gocontext.Context, client *containerd.Client, container containerd.Container, checkpoint string, tty, nullIO bool, opts ...containerd.NewTaskOpts) (containerd.Task, error) {
-	stdio := cio.NewCreator(cio.WithStdio)
+func NewTask(ctx gocontext.Context, client *containerd.Client, container containerd.Container, checkpoint string, tty, nullIO bool, fifoDir string, opts ...containerd.NewTaskOpts) (containerd.Task, error) {
```

Comment:
I don't like changing the function signature to support this functionality.  You should just have this supported under the IO opts